app/src/main/report/actionTypes/ReportsActionsTypes.ts

```diff
@@ -3,6 +3,7 @@ import {Version} from "../../shared/models/reports/Report";
 
 export enum ReportTypeKeys {
     REPORTS_FETCHED = "REPORTS_FETCHED",
+    REPORTS_SORTED = "REPORTS_SORTED",
```

Comment:
For this action I'd probably go with `SORT_REPORTS`, because it's dispatched when the user makes a request to sort reports, not after the reports are sorted?